src/main/java/fdv/controller/SeanceController.java

```diff
@@ -0,0 +1,68 @@
+package fdv.controller;
```

Comment:
Read [Google Code Convention](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names) about packages names (name must have sens, no abbreviation) and read it whole to learn about Java code style